doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -2,8 +2,9 @@
 ## 11.0.0
 - [General] **Attention:** Added native php types for argument types, property types, return types and strict type declaration where possible. 
  This results in **various bc breaks**. Please make sure to add the corresponding types to your implementation.
+- [Symfony] Bumped Symfony packages to "^6.1". Pimcore 11 will only support Symfony 6.
```

Comment:
Note that [Symfony 6.1 is already EOL](https://symfony.com/releases/6.1). Maybe bump directly to 6.2?

Reply:
That's a good point, didn't even realized that 6.1 being EOL and just 3 days ago 😮 Thak you for the heads-up!

We are bumping to 6.2 on the [PR removing SensioFrameworkExtraBundle ](https://github.com/pimcore/pimcore/pull/14062/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34), so i will add the doc change over there before merging it